### PR TITLE
Configure load balancer on a per-cluster and operator-wide level

### PR DESCRIPTION
**Pull request by alexeyklyukin**
- make the load balancer configurable on a per cluster and operator-wide level
- restrict connections to clusters with the load balancers but without allowedSourceRanges defined.

Addresses #54

### CODE REVIEW

File: manifests/testpostgresql.yaml [hunk 1]
```diff
@@ -41,6 +41,7 @@ spec:
     loop_wait: &loop_wait 10
     retry_timeout: 10
     maximum_lag_on_failover: 33554432
+    useLoadBalancer: true
```
Comment (ikitiki) [inline manifests/testpostgresql.yaml]: why camelCase?

File: pkg/cluster/resources.go [hunk 2]
```diff
@@ -249,9 +249,48 @@ func (c *Cluster) updateService(role PostgresRole, newService *v1.Service) error
...
+		if err != nil {
+			return fmt.Errorf("could not create service '%s': '%v'", serviceName, err)
+		}
+		c.Service[role] = svc
+		// recreate the endpoint
+		if role == Master {
+			endpointSpec := c.genMasterEndpoints(currentEndpoint.Subsets)
+			ep, err := c.KubeClient.Endpoints(c.Service[role].Namespace).Create(endpointSpec)
+			if err != nil {
+				return fmt.Errorf("could not create endpoint '%s': '%v'", ep.Name, err)
```
Comment (ikitiki) [inline pkg/cluster/resources.go]: namespace will be missed in the error message

**Comment by ikitiki** [comment]
👍

**Comment by alexeyklyukin** [comment]
👍

**Comment by ikitiki** [comment]
👍

**Pull request merged**